drivers/mfrc522/Makefile.dep

```diff
@@ -0,0 +1,5 @@
+FEATURES_REQUIRED += periph_gpio
```

Comment:
We need the Kconfig file for this

Reply:
`TEST_KCONFIG` is enabled when that is set in the environment and allows to resolve dependencies using Kconfig instead of Makefile.dep. It's there to test the ongoing migration. You can check it by calling `TEST_KCONFIG=1 make menuconfig`.

Reply:
Ah cool, ty 👍